fandogh_cli/__init__.py

```diff
@@ -0,0 +1,2 @@
+VERSION = "0.2.3"
```

Comment:
why do we need these attributes?

Reply:
We need to have access to version number inside the package, I can't read it from `setup.py`

Reply:
Check fandogh —version